lib/Memory/Store.js

```diff
@@ -101,10 +101,11 @@ class MemoryStore extends Store {
    *
    * @public
    * @param {Object} query The query that will be used for fetch the state.
+   * @param {Object} params TODO: quick & dirty, not used but needed by {@link Flux}'s observe method.
```

Comment:
Nop it's just a quick fix in order to use MemoryStore, i will remove this according to the #37